integrations/tekton/README.md

```diff
@@ -0,0 +1,28 @@
+# Tekton Pipeline Example
+
+This example pipeline allows you to easily run an ORT scan on [Tekton](https://tekton.dev/), some knowledge about Tekton will help to understand this example pipeline.. This example pipeline works best on [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine).
+
+## Getting Started
+To run the pipeline, you will need the following things:
+* A Kubernetes Cluster with Tekton installed (follow the [installation instructions](https://tekton.dev/docs/pipelines/install/))
```

Comment:
- s/Cluster/cluster
- In general, please also end sentences from bullet lists with a dot (also in lists below).